### ISSUE TITLE
Swift 4.2 + Xcode 10

### ISSUE THREAD

**Issue by domdop** [bustoutsolutions/siesta#273]
Could you please add support for Swift 4.2 and Xcode 10

Thank you

**Comment by pcantrell** [comment]
Siesta builds and runs just fine with XC10 / Swift 4.2. Is there a specific issue you’re seeing?

**Comment by CodeEagle** [comment]
@pcantrell 
![image](https://user-images.githubusercontent.com/2311352/45855469-d7e04f00-bd82-11e8-97ba-132b1009481f.png)

**Comment by pcantrell** [comment]
It looks like whatever you're compiling it with isn’t respecting the SWIFT_VERSION flag, which should still be set to 4.0 (though it compiles with 4.2).

Hmm, are you getting this error in an app that includes Siesta via CocoaPods, perhaps?

(With all these error reports, don't forget to include at least minimal details about the specific error, context and configuration, etc. It reduces the back-and-forth.)

**Comment by CodeEagle** [comment]
When your Target change to swift4.2, `pod install` without config anything, the pod target also set swift version to 4.2, `pod cache clean --all` first.

**Comment by pcantrell** [comment]
Indeed, it looks like CocoaPods can't mix Swift versions per pod, even though Xcode itself can, and ignored the Swift version setting of pod dependencies:

https://github.com/CocoaPods/CocoaPods/issues/7099
https://github.com/CocoaPods/CocoaPods/issues/6791

**Comment by domdop** [comment]
Ich werde ab  21.09.2018 nicht im Büro sein. Ich kehre zurück am
01.10.2018.
Ich werde Ihre Nachricht nach meiner Rückkehr beantworten.

Die Rückmeldung bezieht sich auf ein Mail mit folgendem Thema:
Re: [bustoutsolutions/siesta] Swift 4.2 + Xcode 10 (#273)

____________________________________________________________________________________________
Gesendet (c) GRZ/RACON Linz 2018 Agent 'Abwesenheit'

----------------------------------------
Raiffeisen Software GmbH
Firmenbuchnummer: 86804d, Landesgericht Linz
DVR: 0572934, UID: ATU 36961105

Der Austausch von Nachrichten mit dem oben angeführten Absender via E-Mail
dient ausschließlich Informationszwecken. Rechtsgeschäftliche Erklärungen
dürfen über dieses Medium nicht ausgetauscht werden.
Correspondence with above mentioned sender via e-mail is only for
information purposes. This medium is not to be used for the exchange of
legally-binding communications.
----------------------------------------
